internal/user/fake.go

```diff
@@ -0,0 +1,18 @@
+package user
+
+func NewFakePrefsInterface() *FakePrefsInterface {
```

Comment:
and likewise `FakePrefsInterface` should just be `FakePrefs`

Reply:
done